### PR TITLE
Preview window

### PR DESCRIPTION
**Pull request by rajvirtual**
Temp hiding of Dashboard/Patients.

### CODE REVIEW

File: src/components/Dashboard/PatientCard/index.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
 import React from 'react';
 import styled from 'styled-components';
-import { useHistory } from 'react-router';
+// import { useHistory } from 'react-router';
```
Comment (nastikue) [inline src/components/Dashboard/PatientCard/index.js]: please remove comments, leave code. As this is temporary change create config prop that will hide necessary parts of the page

File: src/components/Dashboard/PatientCard/index.js [hunk 2]
```diff
@@ -67,16 +67,17 @@ const PatientVitalFieldName = styled.div`
...
+    return; //TOD: Temp
+    // const val = name.replace(/ /g, '_').toLowerCase();
+    // history.push({ pathname: `/patients/${val}`, state: {} });
   };
   return (
     <div className="patient-info">
       {/*patient name & state */}
       <PatientStatusWrp>
-        <State selectedState={patient.status} />
+        {/* <State selectedState={patient.status} /> */}
```
Comment (nastikue) [inline src/components/Dashboard/PatientCard/index.js]:
```suggestion
        {config.isShowSelectedState && <State selectedState={patient.status} />}
```

File: src/components/DesktopPatientTable/index.js [hunk 3]
```diff
@@ -98,10 +98,12 @@ const DesktopPatientTable = (props) => {
           return (
             <InfoAndGraphWrapper key={index} className="mb-0">
               <InfoWrapper>
-                <Status selectedCases={patient.status} />
+                {/* <Status selectedCases={patient.status} /> */}
```
Comment (nastikue) [inline src/components/DesktopPatientTable/index.js]:
```suggestion
                {config.isShowSelectedCases &&  <Status selectedCases={patient.status} />}
```

File: src/components/DesktopPatientTable/index.js [hunk 4]
```diff
@@ -98,10 +98,12 @@ const DesktopPatientTable = (props) => {
...
                 <Link
                   className="card-name patient-link--small min-width-20 mr-2"
-                  to={`/patients/${patient.patientId}/encounter/create`}>
+                  to={''}
+                  // to={`/patients/${patient.patientId}/encounter/create`}>
+                >
```
Comment (nastikue) [inline src/components/DesktopPatientTable/index.js]:
```suggestion
                  to={config.isRedirectToCreateEncounter ? `/patients/${patient.patientId}/encounter/create` : ''}>
```

File: src/pages/Dashboard.js [hunk 5]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
+  // }, [patients, selectedCases]);
 
   return (
     <DashboardLayout>
       <FirstRow>
         <Headings>
           <InfoWrapper>
-            <ViewName>Dashboard</ViewName>
+            <ViewName>Patients vitals</ViewName>
+            {/* <ViewName>Dashboard</ViewName> */}
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
            {config.isShowDashboardName && <ViewName>Dashboard</ViewName>}
```

File: src/pages/Dashboard.js [hunk 6]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
             <DateAndTimeWrap>
               <TimeImage src={time} />
               <DateAndTime>{getDate()}</DateAndTime>
             </DateAndTimeWrap>
           </InfoWrapper>
-          <CasesCardComponent
+          {/* <CasesCardComponent
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
          {config.isShowCasesCard &&  <CasesCardComponent
```

File: src/pages/Dashboard.js [hunk 7]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
             casesCardData={patientRiskData}
             changesCases={changesCases}
             selectedCases={selectedCases}
-          />
-          <SearchWrapper>
+          /> */}
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
          />}
```

File: src/pages/Dashboard.js [hunk 8]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
+          {/* <SearchWrapper>
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
          {config.isShowSearch &&  <SearchWrapper>
```

File: src/pages/Dashboard.js [hunk 9]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
             <CasesHeader>{selectedCases} Cases</CasesHeader>
             <SearchInput
               placeholder="Search"
               searchText={searchText}
               onChange={handleSearchText}
               searchRef={searchRef}
             />
-          </SearchWrapper>
+          </SearchWrapper> */}
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
          </SearchWrapper>}
```

File: src/pages/Dashboard.js [hunk 10]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
         </Headings>
       </FirstRow>
 
       {/* Doctors View - Patients List along with current conditions */}
       <PatientsWrapper>
-        {filteredPatients?.map((patient) => (
+        {patients?.map((patient) => (
```
Comment (nastikue) [inline src/pages/Dashboard.js:180]:
```suggestion
        {filteredPatients?.map((patient) => (
```

File: src/pages/Dashboard.js [hunk 11]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
           <PatientCard key={patient.patientId} patient={patient} />
         ))}
+        {/* {filteredPatients?.map((patient) => (
+          <PatientCard key={patient.patientId} patient={patient} />
+        ))} */}
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
        {config.isShowPatientCards &&  filteredPatients?.map((patient) => (
          <PatientCard key={patient.patientId} patient={patient} />
        ))}
```

File: src/pages/Dashboard.js [hunk 12]
```diff
@@ -83,93 +83,99 @@ const DeskTopViewPatient = styled.section`
...
       </PatientsWrapper>
       <DeskTopViewPatient>
         {patients ? (
           <>
-            <HeaderSearchWrap className="w-100 mb-3">
+            {/* <HeaderSearchWrap className="w-100 mb-3">
+                    <TypeHeader>{selectedCases} Risk Cases</TypeHeader>
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
            {config.isShowSearch &&  <HeaderSearchWrap className="w-100 mb-3">
```

File: src/pages/Dashboard.js [hunk 13]
```diff
@@ -180,10 +186,11 @@ const DashBoardComponent = () => {
                   searchRef={searchRef}
                 />
               </InputContainer>
-            </HeaderSearchWrap>
+            </HeaderSearchWrap> */}
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
            </HeaderSearchWrap>}
```

File: src/pages/Dashboard.js [hunk 14]
```diff
@@ -180,10 +186,11 @@ const DashBoardComponent = () => {
...
             <DesktopPatientTable
-              selectedCaseData={filteredPatients}
-              selectedCases={selectedCases}
+              selectedCaseData={patients}
+              // selectedCaseData={filteredPatients}
+              // selectedCases={selectedCases}
```
Comment (nastikue) [inline src/pages/Dashboard.js]:
```suggestion
              selectedCaseData={config.isShowCasesCard ?  filteredPatients : undefined}
              selectedCases={config.isShowCasesCard ?  selectedCases : undefined}
```

**Comment by nastikue** [comment]
@rajvirtual please make all hides by config, not with comments. Otherwise it would be very hard to get working code if smth is changed in any of the components before code is uncommented

File: src/components/Dashboard/PatientCard/index.js (hunk 1 above)
Comment (rajvirtual) [reply src/components/Dashboard/PatientCard/index.js]: we cannot leave unused Code. commit will fail (Lint)

**Pull request merged**